packages/kununu-footer/index.scss

```diff
@@ -116,6 +118,27 @@
   }
 }
 
+.title {
+  color: $gray-base-40;
+  margin: 0;
+  padding: 0;
+  letter-spacing: 0;
+  line-height: $line-height-h3;
+  font-weight: 600;
+  display: block;
+  font-size: 18px;
+  margin-bottom: $baseline-grid * 2;
+  margin-top: $baseline-grid * 3;
+
+  @media (max-width: $screen-sm-max) {
```

Comment:
Should we follow a "mobile first" approach?

Reply:
Yes, I suppose it could use a refactor to mobile first. I will update this.